src/internal/HtmlColor.h

```diff
@@ -76,6 +90,23 @@ struct HtmlColor
     };
 
     // ------------------------------------------------------------------------
+    // Parse a HTML4/CSS3 color name
+    //
+    // name - the color name
+    //
+    // It accepts all standard HTML4 names and, if USE_CSS3_COLORS macro is
+    // defined, the color names defined in CSS3 also
+    // ------------------------------------------------------------------------
+    bool Parse(const char* name);
+
+    // ------------------------------------------------------------------------
+    // Converts this color code to its HTML4/CSS3 name
+    //
+    // name - array of MAX_HTML_COLOR_NAME_LEN character to store the name
+    // ------------------------------------------------------------------------
+    void ToString(char name[MAX_HTML_COLOR_NAME_LEN]) const;
```

Comment:
generally for arguments to a call that fills in a char array, you want to pass a pointer to the array and a size of that array; and then fill it with as many chars as you have (plus a null terminator) or the size; which ever is less; returning the number of chars written.
```uint8_t ToString(char* buffer, uint8_t bufferSize);```
Note:  the term size has a specific connotation; its the size of the buffer, not the length of string it can hold, length would be one less (have to store a null terminator)